src/ast/dibuilderbpf.h

```diff
@@ -14,17 +23,42 @@ class DIBuilderBPF : public DIBuilder
 
   void createFunctionDebugInfo(Function &func);
 
+  DIType *getInt8Ty();
+  DIType *getInt16Ty();
+  DIType *getInt32Ty();
   DIType *getInt64Ty();
+  DIType *getInt128Ty();
   DIType *getInt8PtrTy();
+  DIType *getIntTy();
+
+  DIType *GetType(const SizedType &stype);
+  DIType *CreateTupleType(const SizedType &stype);
+  DIType *createPointerMemberType(const std::string &name,
+                                  uint64_t offset,
+                                  DIType *type);
+  DIType *GetMapKeyType(const MapKey &key, const SizedType &value_type);
+  DIType *GetMapFieldInt(int value);
+  DIGlobalVariableExpression *createMapEntry(const std::string &name,
+                                             libbpf::bpf_map_type map_type,
+                                             uint64_t max_entries,
+                                             const MapKey &key,
+                                             const SizedType &value_type);
 
   DIFile *file = nullptr;
 
 private:
   struct
   {
+    DIType *int8 = nullptr;
+    DIType *int16 = nullptr;
+    DIType *int32 = nullptr;
     DIType *int64 = nullptr;
+    DIType *int128 = nullptr;
     DIType *int8_ptr = nullptr;
+    DIType *int_ = nullptr;
```

Comment:
Why the need for this private `int_` rather than just using `int32`?

Reply:
Hm, good question. This is used for entries for integer fields of a map entry (`type` and `max_entries`) which libbpf expects to be pointers to an array of `int` (not `int32`). It's possible that it would work with `int32`, too, but I was trying to stay as close to libbpf's behaviour as possible.